CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * Allows to disable page refresh on content changed for page types.
 * Adds prop to `AposInputMixin` to disable blur emit.
 * Adds `throttle` function in ui module utils.
+* Adds a `publicBundle` option to `@apostrophecms/asset` - when set to `false`, it will prevent from rebuilding asset bundles unless `APOS_DEV` env variable is set or when there is no bundle already built - to be used when using Apostrophe with an External Front End
```

Comment:
I suggest "will prevent the rebuilding of the `ui/src` public bundle for non-admin use"

And it's fine to add "such as Astro" as that's the only implementation at the moment and will clarify the utility for most